platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/state/DefaultStateManagementComponent.java

```diff
@@ -178,7 +179,7 @@ public DefaultStateManagementComponent(
             @NonNull final IssConsumer issConsumer,
             @NonNull final HaltRequestedConsumer haltRequestedConsumer,
             @NonNull final FatalErrorConsumer fatalErrorConsumer,
-            @NonNull final PreConsensusEventWriter preConsensusEventWriter,
+            @NonNull final PreconsensusEventWriter preConsensusEventWriter,
```

Comment:
Done. I did not change variations of preConsensus/pre-consensus for things unrelated to the preconsensus event stream... this can happen at a later date.